plugins/ProfessionalServices/tests/Unit/PromoWidgetApplicableTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\ProfessionalServices\tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Piwik\Plugin\Manager;
+use Piwik\Plugins\ProfessionalServices\PromoWidgetApplicable;
+use Piwik\ProfessionalServices\Advertising;
+
+class PromoWidgetApplicableTest extends TestCase
```

Comment:
It can be helpful to add new tests into a suitably name group so they can be explicitly referenced when testing, but less of an issue when the plugin has few tests. Not important.

```suggestion
/**
 * @group PromoWidgetApplicable
 */
class PromoWidgetApplicableTest extends TestCase
```